pages/common/fish.md

```diff
@@ -23,10 +27,6 @@
 
 `fish --private`
 
-- Display version information and exit:
+- Print the version:
 
 `fish --version`
-
-- Set and export environmental variables that persist across shell restarts (from within the shell only):
-
-`set -Ux {{variable_name}} {{variable_value}}`
```

Comment:
I'm not sure if we should remove this. This specific command is tightly connected to the shell and very useful. We cannot really document it in `set.md`, because it's a shell builtin in this case.

Reply:
I can leave this example here also because there are less than 8 examples now. 😅 I agree, it's useful and I think we can add to another tldr pages builtins useful samples such as this one.